docker/docker-jupyter/jupyter_notebook_config.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import os
+from IPython.lib import passwd
+
+#c = c  # pylint:disable=undefined-variable
+c = get_config()
```

Comment:
where is `get_config()` coming from?

Reply:
In [Overview of the IPython configuration system](https://ipython.org/ipython-doc/3/development/config.html), it says:
> While in a configuration file, to get a reference to this object, simply call the get_config() function, which is available in the global namespace of the script.